waku/factory/internal_config.nim

```diff
@@ -36,6 +36,8 @@ proc enrConfiguration*(
       let shardsRes = topicsToRelayShards(conf.pubsubTopics)
       if shardsRes.isOk() and shardsRes.get().isSome():
         shardsLocal = shardsRes.get().get().shardIds
+      elif not shardsRes.get().isNone():
+        info "No pubsub topics specified or pubsubtopic is of type Named sharding "
```

Comment:
nitpick comment: I have the impression that we use lowercase when starting log messages